.github/workflows/linux-build-and-test.yaml

```diff
@@ -71,12 +71,7 @@ jobs:
       run: |
         brew install gawk
 
-        # Force use of OpenSSL 3.1.4, since OpenSSL 3.2.0 crashes with recent
-        # PostgreSQL versions on OS X (see https://github.com/Homebrew/homebrew-core/issues/155651)
         brew install --overwrite python@3.12
-        brew unlink openssl@3
```

Comment:
It seems removing MN makes the OpenSSL downgrade superfluous. Last time, only the 2.13.x release branch was affected and main was fine after the MN code was removed.

Line 74 `brew install --overwrite python@3.12` could also be removed if no downgrade is needed anymore. 

Could you also push your changes to the `prerelase_test` branch? It would be good to check that all CI checks run without any problems and that the upload to the statistics database works. This step is currently skipped since this is PR opened from a fork (I talked with @antekresic about this. The upload was also affected on main last time.).

Reply:
Removed the python line as well and created a PR against the `prerelease_test` branch as well @jnidzwetzki https://github.com/timescale/timescaledb/pull/6581